### ISSUE TITLE
remove specialization/generalization?

### ISSUE THREAD

**Issue by VladimirAlexiev** [qudt/qudt-public-repo#719]
Two props are declared to make a QuantityKind hierarchy: `qudt:generalization/specialization`:
https://github.com/qudt/qudt-public-repo/blob/ac7111711c5e2c891726d3fa0aaa8b90747b4888/schema/SCHEMA_QUDT-v2.1.ttl#L2368-L2377

However, they are not actually used. `skos:broader` is used Instead of `generalization`. For example
[VOCAB_QUDT-QUANTITY-KINDS-ALL-v2.1.ttl#L42](https://github.com/qudt/qudt-public-repo/blob/ac7111711c5e2c891726d3fa0aaa8b90747b4888/vocab/quantitykinds/VOCAB_QUDT-QUANTITY-KINDS-ALL-v2.1.ttl#L42):
```ttl
quantitykind:AbsoluteActivity  a qudt:QuantityKind ;
  skos:broader quantitykind:InverseVolume 
```

https://github.com/qudt/qudt-public-repo/pull/57 proposes to relate the two props, but it has not been taken up:
```ttl
qudt:generalization rdfs:subPropertyOf skos:broader
```

For avoidance of doubt, I propose to remove `qudt:specialization/generalization`. 
Then these lines need to be removed:
https://github.com/qudt/qudt-public-repo/blob/ac7111711c5e2c891726d3fa0aaa8b90747b4888/schema/SCHEMA_QUDT-v2.1.ttl#L1048-L1052
As well as:
https://github.com/qudt/qudt-public-repo/blob/ac7111711c5e2c891726d3fa0aaa8b90747b4888/schema/SCHEMA_QUDT-v2.1.ttl#L1068-L1072

This restriction allows `qudt:QuantityKind` to use `skos:broader`:
https://github.com/qudt/qudt-public-repo/blob/ac7111711c5e2c891726d3fa0aaa8b90747b4888/schema/SCHEMA_QUDT-v2.1.ttl#L90-L94

The SKOS ontology declares the domain and range of  `skos:broader` to be `skos:Concept` so under RDFS reasoning each QuantityKind that participates in the hierarchy will be inferred to be `skos:Concept` .
However, I think we can leave this "issue" to the consumer: if they don't want that, they should not import SKOS.

**Comment by steveraysteveray** [comment]
I agree with your suggestions!

**Comment by steveraysteveray** [comment]
Resolved in PR #721
